src/persist-client/src/fetch.rs

```diff
@@ -44,6 +48,30 @@ use crate::internal::state::{BatchPart, HollowBatchPart};
 use crate::read::LeasedReaderId;
 use crate::ShardId;
 
+/// The columnar format we write down a batch with.
+pub const PART_DECODE_FORMAT: Config<&'static str> = Config::new(
+    "persist_part_decode_format",
+    PartDecodeFormat::default().as_str(),
+    "Format we'll use to decode a Persist 'Part' (Materialize).",
```

Comment:
Same comment as the other config - it would be helpful to mention the valid values here.

Reply:
Added